### DISCUSSION TITLE
Get Raw Values for Calibration Using RobotInterface

### DISCUSSION THREAD

**Discussion by lschilli** [robotology/community#17] [Q&A] [answered]
I recently checked if I can perform finger calibration and arm / arm fine calibration using the robotInterface. It seems I am missing how to start the robotInterface with a "raw" configuration to be able to read values from the robotMotorGui as described here:
http://robotcub.svn.sourceforge.net/svnroot/robotcub/trunk/iCubPlatform/doc/manuals/ST2009_RBCS_ICUB_ARM_HAND_CALIBRATION_01.doc
(linked from here: http://wiki.icub.org/wiki/Manual)
So the question is: Using the robotInterface, how can the values that need to be entered in the calibration xls file be determined.

**Comment by randaz81** [comment, marked as answer]
Hi, indeed the calibration procedure described in that document refers to the old iCubInterface executable. With robotInterface, the procedure changed a little bit: you don't need anymore to start the robot with a _raw_.ini file. Instead, you just need to edit your general.xml file and set the parameters 'skipCalibration' and 'useRawEncoderData' to true. During the launch of robotInterface, you'll be asked multiple times (one time per part) to press enter and verify that your fault button is pressed. Then you'll be able to take the raw values from the robotMotorGui and put them in the excel file as usual.

**Comment by lschilli** [comment]
Thank you. This worked.
